### PR TITLE
Added some Details

### PR DESCRIPTION
**Pull request by khatabwedaa**
- Added categories
- Added CC0 1.0 Universal LICENSE
- Added some Contributing Guide, please check it and change it if you want :smile: :v:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,21 @@
 This repo aims to collect OSS made in Sudan or by Sudanese.
 Please feel free to submit a PR with your OSS (Package, App ...etc.), or OSS from someone you know.
 
-# Template:
+> For Contribute read [Contributing Guide](contributing.md) please
 
-- [Open Source Project Name](https://github.com/obiefy/api-response) - Small Description about the Open Source Project.
+## Plugins
```
Comment (obiefy) [inline README.md]: Please rename Plugins to packages. Or Add Plugins and Packages as separated categories.

File: README.md [hunk 2]
```diff
@@ -2,15 +2,21 @@
...
 - [Meraki UI](https://github.com/merakiui/website) - Beautiful Tailwindcss components that support RTL out of the box & fully responsive.
 - [Tailwind Copy](https://github.com/the94air/twcopy) - Easily copy Tailwindcss colors and shadows.
 - [Tailwind Builder](https://github.com/Miaababikir/Tailwind-Builder) - Helps you to scaffold your forms structure without the need to repeat yourself.
-- [JoeVue](https://github.com/Miaababikir/JoeVue) - CLI tool help you create your vue file easily.
-- [Odoo-Intellij Plugin](https://github.com/firebitsnet/odoo-intellij) - Its a JetBrains's IDE Plugin that adds Support for developing Odoo applications and modules with IntelliJ IDEA and PyCharm IDEs.
+
+## IDE Extensions
+
+- [Odoo-Intellij Extension](https://github.com/firebitsnet/odoo-intellij) - Its a JetBrains's IDE Plugin that adds Support for developing Odoo applications and modules with IntelliJ IDEA and PyCharm IDEs.
+- [Laravel-snip Extension](https://marketplace.visualstudio.com/items?itemName=khatabwedaa.laravel-snip) - Simple vscode Extension for Laravel routes.
```
Comment (obiefy) [inline README.md]: Kindly update the link to the Open Source repo :)

**Review by obiefy — CHANGES_REQUESTED** [review]
Kindly update the mentioned notes.
Thanks in advance :)

File: README.md (hunk 2 above)
Comment (khatabwedaa) [reply README.md]: i deleted this repo :laughing: but i will remove it :+1:

File: README.md (hunk 1 above)
Comment (khatabwedaa) [reply README.md]: Done :v:

**Comment by obiefy** [comment]
Thanks @khatabwedaa for this PR :)

**Pull request merged**